pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/mapper/SegmentCreationMapper.java

```diff
@@ -235,6 +238,12 @@ protected void map(LongWritable key, Text value, Context context)
   }
 
   protected FileFormat getFileFormat(String fileName) {
+    // ORC files do not necessarily have the .orc file extension nor is there a reliable way to decide
```

Comment:
We don't need this anymore, right?

Reply:
changed the file formatting to bring in a general option